src/components/ToggleSwitch/theme.ts

```diff
@@ -10,7 +10,7 @@ export const toggleSwitchTheme: FlowbiteToggleSwitchTheme = {
     label: 'ml-3 text-sm font-medium text-gray-900 dark:text-gray-300',
   },
   toggle: {
-    base: 'toggle-bg rounded-full border group-focus:ring-4 group-focus:ring-cyan-500/25',
+    base: 'relative shrink-0 toggle-bg rounded-full border group-focus:ring-4 group-focus:ring-cyan-500/25',
```

Comment:
without flex-shrink it will be broken (even with `min-width` values)
![image](https://github.com/themesberg/flowbite-react/assets/73419198/67859c7f-513b-4c6d-bd31-feb0c55d59f7)